README.md

```diff
@@ -36,9 +36,11 @@ A [Durham Expunction and Restoration (DEAR)](https://www.deardurham.org)
 git clone git@github.com:deardurham/dear-petition.git
 cd dear-petition
 docker-compose up -d django
+docker-compose run --rm django python manage.py migrate
 docker-compose run --rm django python manage.py createsuperuser
 ```
 
+When asked for a username and password, enter values of your choosing.  Email address may be left empty.
```

Comment:
I think this can be removed from the `Quick Start` section, as we already have the same listed in the ` Initial Setup` section

Reply:
I'll remove it.  I was conflicted both by repeating it and by not having it both places.